src/Cargo.lock

```diff
@@ -1680,6 +1680,15 @@ dependencies = [
  "serde_derive 1.0.40 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "rls-data"
+version = "0.16.0"
```

Comment:
This looks maybe a bit wrong? A version of `rls-data` is added, but the old one isn't removed. What's happening here?

Reply:
The old version is still used by the RLS, it will get removed when we update the RLS. But we'll have to do that in a separate PR (in any case the RLS is broken right now, so it doesn't make much difference).